sync/syncer.go

```diff
@@ -490,15 +491,21 @@ func (s *Syncer) syncAtxs(atxIds []types.AtxId) ([]*types.ActivationTx, error) {
 
 	//map and sort atxs
 	if len(missingInDb) > 0 {
-
 		output := s.fetchWithFactory(NewNeighborhoodWorker(s, 1, ATxReqFactory(missingInDb)))
 		for out := range output {
 			atxs := out.([]types.ActivationTx)
 			for _, atx := range atxs {
+				if err := s.EnsurePoetProof(atx.GetPoetProofRef()); err != nil {
+					s.Warning("received atx (%v) with syntactically invalid or missing PoET proof (%x): %v",
```

Comment:
should be an error

Reply:
In general, I don't think we should error outcomes that got triggered from peers on the network.

Reply:
I agree with @moshababo's logic that errors should be reserved for things that go wrong within the node, rather than other nodes misbehaving, which should be logged as warnings.

Also worth pointing out that this same code in `handleGossipAtx` is logging warnings:
https://github.com/spacemeshos/go-spacemesh/blob/2a6d7f7cf3aa7fd4104fc5839b7f4bfc97df80ce/miner/block_builder.go#L300-L311

Reply:
Sorry guys, I 've to disagree.
When we notice an abnormal behavior (whether it was caused by the local node or by a remote node), and we don't want to miss this behavior we must print an error.
Like in this case.